src/controller/recordController.js

```diff
@@ -0,0 +1,62 @@
+const Record = require('../models/Record');
+
+const getRecords = async (req, res, next) => {
+  try {
+    const allRecords = await Record.find().sort('title');
+    res.json(allRecords);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// for testing
+const getRecord = async (req, res, next) => {
+  const { id } = req.params; // tbd
+  try {
+    const record = await Record.findById(id);
+    res.json(record);
+  } catch (err) {
+    next(err);
+  }
+};
+
+//for testing
+const addRecord = async (req, res, next) => {
+  const recordInfo = req.body;
+  try {
+    const newRecord = await Record.create(recordInfo);
+    res.json(newRecord);
+  } catch (err) {
+    next(err);
+  }
+};
+
+//not necessary till now
+// const updateRecord = async (req, res, next) => {
+//   const { id } = req.params;
+//   try {
+//     const UpdatedRecord = await Record.findByIdAndUpdate(id, req.body, {
+//       new: true,
+//     });
+//     res.json(UpdatedRecord);
+//   } catch (err) {
+//     next(err);
+//   }
+// };
```

Comment:
Could we avoid pushing commented code to main? You can have things to avoid something breaking, but commented code is never good.

Reply:
removed